<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelVisualizationDockWidget</class>
 <widget class="QDockWidget" name="ChannelVisualizationDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>145</height>
   </rect>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QRadioButton" name="AllChannelsBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>98</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>All Channels</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="OneChannelBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>98</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>One Channel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ChannelComboBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>161</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OneChannelBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>ChannelComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
